index.html

```diff
@@ -741,12 +741,13 @@ <h4>
       <h2>
         Navigation scope
       </h2>
-      <p>
+      <p data-link-for="WebAppManifest">
         A <dfn>navigation scope</dfn> is a [[!URL]] that represents the set of
         URLs to which an <a>application context</a> can be navigated while the
-        manifest is <a>applied</a>. To determine if a URL is within the
-        <a>navigation scope</a>, the user agent MUST run the <a>within
-        scope</a> algorithm.
+        manifest is <a>applied</a>. The <a>navigation scope</a> of a manifest
+        <var>manifest</var> is <var>manifest</var>["<a>scope</a>"]. If this is
+        <code>undefined</code> (because of an error or it was not declared in
```

Comment:
due to an error, or because it was not declared in the manifest

Reply:
Done.